one/src/daemon.rs

```diff
@@ -139,23 +139,35 @@ pub struct DaemonOpts {
         )]
     cors_allow_origins: Vec<String>,
 
-    /// Enable experimental feature flags
+    /// Allow experimental features
     #[arg(
         long,
-        use_value_delimiter = true,
-        value_delimiter = ',',
-        env = "CERAMIC_ONE_EXPERIMENTAL_FEATURE_FLAGS"
+        default_value_t = false,
+        env = "CERAMIC_ONE_EXPERIMENTAL_FEATURES"
     )]
-    experimental_feature_flags: Vec<ExperimentalFeatureFlags>,
+    experimental_features: bool,
+
+    /// Allow deprecated features
+    #[arg(long, default_value_t = false, env = "CERAMIC_ONE_DEPRECATED_FEATURES")]
+    deprecated_features: bool,
 
-    /// Enable feature flags
+    /// Enable authentication; Requires using the experimental-features flag
     #[arg(
         long,
-        use_value_delimiter = true,
-        value_delimiter = ',',
-        env = "CERAMIC_ONE_FEATURE_FLAGS"
+        default_value_t = false,
+        env = "CERAMIC_ONE_AUTHENTICATION",
```

Comment:
huh, I know this isn't really the focus of this PR, but why is this flag called "authentication"?  Isn't this about enabling the access control experiment that @dbcfd was working on?  That's more about access control / authorization than authentication, right?